assets/styles/layouts/_top-nav.scss

```diff
@@ -75,7 +75,7 @@
     }
 
     &:after {
-      content: '\e917';
+      content: "\e918";
```

Comment:
'' vs. "" in CSS

Reply:
Question or statement? I changed this to double quotes to match the content properties everywhere else . It doesn't make a difference in how the CSS is parsed.